models/spleen_deepedit_annotation/configs/inference.json

```diff
@@ -19,7 +19,7 @@
     ],
     "number_intensity_ch": 1,
     "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
-    "use_click": false,
+    "use_click": true,
     "network_def": {
```

Comment:
Hi @Nic-Ma , here I decided to modify the default value rather than remove the normal transforms. The main purpose is to keep an option for users to skip the click related transforms for acceleration.

Reply:
Hi @yiheng-wang-nv , what's the inference speed benchmark with / without click points?
If no obvious difference, let's remove this arg.

Thanks.

Reply:
Thanks @Nic-Ma . I just did experiments on spleen dataset and confirmed that:
1. inference times on w/o click transforms are similar
2. inference results are the same (will not input any click points if with click transforms)

I will remove the arg.

Reply:
Hi @Nic-Ma , I just push a new commit, could you help to re-review it? Thanks!